### PR TITLE
reuse KUBE_WAIT_FOR_LIVE in KUBE_WAIT_FOR_PREREQ

### PR DESCRIPTION
**Pull request by grosser**
followup for https://github.com/zendesk/samson/pull/2996#discussion_r224888225

@rata @adammw

### CODE REVIEW

**Comment by adammw** [comment]
why not specify them in seconds rather than the weird minutes conversion?

**Comment by grosser** [comment]
updated ... @rata new env vars now use seconds

File: plugins/kubernetes/app/models/kubernetes/deploy_executor.rb [hunk 1]
```diff
@@ -5,8 +5,11 @@
 
 module Kubernetes
   class DeployExecutor
-    WAIT_FOR_LIVE = ENV.fetch('KUBE_WAIT_FOR_LIVE', 10).to_i.minutes
-    WAIT_FOR_PREREQ = ENV.fetch('KUBE_WAIT_FOR_PREREQ', 10).to_i.minutes
+    if ENV['KUBE_WAIT_FOR_LIVE'] && !ENV["KUBERNETES_WAIT_FOR_LIVE"]
+      raise "Use KUBERNETES_WAIT_FOR_LIVE with seconds instead of KUBE_WAIT_FOR_LIVE"
+    end
+    WAIT_FOR_LIVE = Integer(ENV.fetch('KUBERNETES_WAIT_FOR_LIVE', '600'))
```
Comment (adammw) [inline plugins/kubernetes/app/models/kubernetes/deploy_executor.rb]: 600 doesn't need to be a string

**Review by adammw — APPROVED** [review]

File: plugins/kubernetes/app/models/kubernetes/deploy_executor.rb (hunk 1 above)
Comment (grosser) [reply plugins/kubernetes/app/models/kubernetes/deploy_executor.rb]: for consistency

File: plugins/kubernetes/app/models/kubernetes/deploy_executor.rb [hunk 2]
```diff
@@ -5,8 +5,11 @@
 
 module Kubernetes
   class DeployExecutor
-    WAIT_FOR_LIVE = ENV.fetch('KUBE_WAIT_FOR_LIVE', 10).to_i.minutes
-    WAIT_FOR_PREREQ = ENV.fetch('KUBE_WAIT_FOR_PREREQ', 10).to_i.minutes
+    if ENV['KUBE_WAIT_FOR_LIVE'] && !ENV["KUBERNETES_WAIT_FOR_LIVE"]
+      raise "Use KUBERNETES_WAIT_FOR_LIVE with seconds instead of KUBE_WAIT_FOR_LIVE"
```
Comment (rata) [inline plugins/kubernetes/app/models/kubernetes/deploy_executor.rb]: LOL, nice way to force all that were using the variable to update :-D

**Comment by rata** [comment]
@grosser thanks! :)

**Pull request merged**